demos/browser/app/messagingSession/messagingSession.ts

```diff
@@ -57,7 +59,11 @@ export class DemoMessagingSessionApp implements MessagingSessionObserver {
         const endpoint = await chime.send(new GetMessagingSessionEndpointCommand());
         this.userArn = (document.getElementById('userArn') as HTMLInputElement).value;
         this.sessionId = (document.getElementById('sessionId') as HTMLInputElement).value;
+        this.prefetchOn = (document.getElementById('prefetchOn') as HTMLInputElement).checked;
         this.configuration = new MessagingSessionConfiguration(this.userArn, this.sessionId, endpoint.Endpoint.Url, chime);
+        if (this.prefetchOn) {
+          this.configuration.prefetchOn = PrefetchOn.Connect;
```

Comment:
nit: you can just write `this.configuration.prefetchOn = (document.getElementById('prefetchOn') as HTMLInputElement).checked;`